### PR TITLE
Feature/cbi 2110 download wasm artifact

### PR DESCRIPTION
**Pull request by MRamanenkau**
https://cerenetwork.atlassian.net/browse/CBI-2110

You can check the artifact [here](https://github.com/Cerebellum-Network/pos-network-node/actions/runs/2791581327)

### CODE REVIEW

File: .github/workflows/build-and-push-docker-image.yml [hunk 1]
```diff
@@ -27,8 +27,19 @@ jobs:
...
           docker push $ECR_REGISTRY/$ECR_REPOSITORY:latest
+      - name: Copy wasm artifact from the image
+        run: |
+          image_id="${{ env.image_id }}"
+          container_id=$(docker create $image_id)
+          docker cp "${container_id}:/home/cere/node-runtime-artifacts/node_runtime.compact.wasm" ./
+      - name: Upload wasm artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: node_runtime.compact.wasm
```
Comment (AndreiNavoichyk) [inline .github/workflows/build-and-push-docker-image.yml]: Let's add a unique identifier to the artifact filename. Use GITHUB_SHA

Comment (MRamanenkau) [reply .github/workflows/build-and-push-docker-image.yml]: Added. Here is an [example](https://github.com/Cerebellum-Network/pos-network-node/actions/runs/2795433038)

File: .github/workflows/build-and-push-docker-image.yml [hunk 2]
```diff
@@ -27,8 +27,20 @@ jobs:
...
+          echo IMAGE_ID=$image_id >> $GITHUB_ENV
           echo "::set-output name=image::$ECR_REGISTRY/$ECR_REPOSITORY:$GITHUB_SHA"
           docker image tag $ECR_REGISTRY/$ECR_REPOSITORY:$GITHUB_SHA $ECR_REGISTRY/$ECR_REPOSITORY:latest
           docker push $ECR_REGISTRY/$ECR_REPOSITORY:$GITHUB_SHA
           docker push $ECR_REGISTRY/$ECR_REPOSITORY:latest
+      - name: Copy wasm artifact from the image
+        run: |
+          container_id=$(docker create ${{ env.IMAGE_ID }})
+          docker cp $container_id:/home/cere/node-runtime-artifacts/node_runtime.compact.wasm ./
+          short_github_sha=${GITHUB_SHA:0:9}
```
Comment (AndreiNavoichyk) [inline .github/workflows/build-and-push-docker-image.yml]: Why 9?

**Review by AndreiNavoichyk — DISMISSED** [review]

File: .github/workflows/build-and-push-docker-image.yml (hunk 2 above)
Comment (MRamanenkau) [reply .github/workflows/build-and-push-docker-image.yml]: I decided to stick to the length git uses (image). But now I see that the usual length is 7 even for git. Updated to 7.
Applications

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**